java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -45,11 +52,23 @@ public class HttpConsumer {
     private ScheduledExecutorService executorService;
     private Tracer tracer;
     private CountDownLatch messagesReceivedLatch;
+    private static boolean enableAutoCommit = false;
```

Comment:
You don't need this field to be static. I see you are actually reading it in the main, so the reason to make it static but it's not needed.
In the end, it has just to be a non static field of the HttpConsumer class which gets it from the config (and related properties collection). This field is used in all instance methods of this class, there is no usage in the main so no need to write it there.

Reply:
Fixed